regexes/device/mobiles.yml

```diff
@@ -2155,17 +2180,17 @@ Kazam:
 
 # Kiano
 Kiano:
-  regex: 'Kiano|Elegance_5_0|CORE 10.1 DUAL 3G|Intelect103G|SlimTab7[_ ]3GR[);/ ]'
+  regex: 'Kiano|Elegance_5_0|CORE 10.1 DUAL 3G|Intelect ?(7|8|10)|Slim ?Tab ?\d+(?:[_ ]3GR?)?[);/ ]'
   device: 'smartphone'
   models:
-    - regex: 'SlimTab7[_ ]3GR'
-      model: 'SlimTab7 GR'
+    - regex: 'Slim ?Tab ?(\d+)(?:[_ ]3GR?)?'
+      model: 'Slim Tab $1'
```

Comment:
Don't think it's important. Maybe we could/should remove that in some other places, as well